src/main/java/hillel/spring/doctors/DoctorService.java

```diff
@@ -82,7 +84,7 @@ public Appointment findOrCreateSchedule(Integer doctorId, LocalDate date) {
     }
 
     public void schedulePetToDoctor(Integer doctorId, LocalDate date, Integer hour, Integer petId) {
```

Comment:
What if two different people simultaneously attempt to schedule an appointment to the **same** doctor on **different** time?
Try to emulate this by adding `Thread.sleep()` in the service method.